STM32F405RGTx/Core/Inc/svpwm_thread.hpp

```diff
@@ -4,11 +4,23 @@
 #include "main.h"
 #include "util.hpp"
 
-#ifndef INC_SVPWM_H_
-#define INC_SVPWM_H_
+#define MODINDEX        0.9             // Set ModIndex here        
+#define FREQUENCY       1               // Set Frequency here (kHz)
+#define	PI_THIRD		1.04719755119660f

```

Comment:
just letting u know you can do pi/3 here and the compiler will optimize that into the actual value instead of evaluating the thing every time